hazelcast/src/main/java/com/hazelcast/map/EntryProcessor.java

```diff
@@ -129,7 +129,7 @@ public interface EntryProcessor<K, V, R> extends Serializable {
     /**
      * Get the entry processor to be applied to backup entries.
      * <p>
-     * In case of a readonly execution, {@code null} can be returned to indicate
+     * In case of a {@link ReadOnly} execution, {@code null} can be returned to indicate
```

Comment:
I think that a link to the `ReadOnly` interface here indicates that this sentence only applies to `EntryProcessor`s that implement the `ReadOnly` interface. I don't think this is true (ie. I think that if I return `null` backup processor from any `EntryProcessor` that does not implement `ReadOnly` still means no backup will be executed), so I wouldn't change this sentence.

Reply:
04743ba561fdf2d294deee010a5f8a65248ed976